src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -323,6 +324,26 @@ public function xdebugAction(): Response
         return new Response($xdebugInfo, 200, ['Content-Type' => 'text/html']);
     }
 
+    /**
+     * Return the JetBrainMono font.
```

Comment:
I'd make this comment more generic: "Downloads the custom web fonts used in the profiler"